src/box/lua/config/init.lua

```diff
@@ -150,6 +150,7 @@ function methods._initialize(self)
     self:_register_applier(require('internal.config.applier.console'))
     self:_register_applier(require('internal.config.applier.fiber'))
     self:_register_applier(require('internal.config.applier.app'))
+    self:_register_applier(require('internal.config.applier.sharding'))
```

Comment:
We should configure sharding first and only them call the user provided code (application).

(I know, the `app` applier actually performs all its logic in the `post_apply()` hook, but I would anyway place `app` at the last position.)

Reply:
True, fixed.